### ISSUE TITLE
still crashes

### ISSUE THREAD

**Issue by aerith1984** [PsykoDev/fly-forever#6]
still crashes

**Comment by PsykoDev** [comment]
normaly it's good now

**Comment by aerith1984** [comment]
tried it after your last update works as intended 20 seconds after gauce is emptied it crashes for me

**Comment by PsykoDev** [comment]
what's mount you use ?

**Comment by aerith1984** [comment]
tera club mount i used when it was crashing..cos it was crushing i deleted it till u fix and sent u issue

**Comment by aerith1984** [comment]
no crashes on 1.5 dragon mount though..just tested

**Comment by PsykoDev** [comment]
yeah Tera club mount is different than other " normal " mount i can't fix it for Tera club

**Comment by aerith1984** [comment]
well i dont but still crashes for me now with tulpar mount like i told u after 20-30 secs after bar is emptied

**Comment by PsykoDev** [comment]
have you download my latest def file / opcode / nodejs ?

**Comment by PsykoDev** [comment]
https://github.com/PsykoDev/Full-Opcode-Tera-EU-NA
https://github.com/PsykoDev/Last-Def-Files
https://nodejs.org/dist/v15.12.0/node-v15.12.0-x64.msi

**Comment by aerith1984** [comment]
yes i have everything..still crashes

**Comment by Ganondorf91** [comment]
yup, same here... i thought this new update would fix the problem but once your drain all the bar the game will crash randomly after, unless you go back to the ground before that happens and yea i tried various mounts

**Comment by PsykoDev** [comment]
hum it's rly weird i've try a lot of time with much mounts and never game crash, i'll check that soon
